devtools/ci/benchmark.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SCRIPT_PATH="$GITHUB_WORKSPACE/ckb-integration-test/ckb-bench"
```

Comment:
Naming `CKB_BENCH_PATH` is more accurate.
`SCRIPT_PATH="${CKB_BENCH_PATH}/script"`

Reply:
done